src/main/java/Main.java

```diff
@@ -26,6 +40,26 @@ public void start(Stage primaryStage) throws Exception{
         tt.setAutoReverse(true);
         tt.setCycleCount(99);
         //tt.play();
+
+        api.getRants(Sort.ALGO, 10, 0).thenAcceptAsync(results -> {
```

Comment:
Ah OK for supply and then accept confused, thought then accept her to be async as well.

This was just for testing the PostControl and highlight the issue I'm having with the image